spec/features/create_pdf_spec.rb

```diff
@@ -22,6 +22,8 @@
       # Hyrax::BasicMetadata attributes that we don't want in the form
       expect(page).to have_no_content('Keywords')
       expect(page).to have_no_content('Location')
+      # We don't want the transcript UI to show up on the Pdf form
+      expect(page).not_to have_content('You will need to attach an XML file to to this work to select a transcript.')
```

Comment:
@no-reply this replaces the view spec for forms that shouldn't have the transcript UI